### ISSUE TITLE
problems running kokkos on Crusher

### ISSUE THREAD

**Issue by Trzs** [cctbx/cctbx_project#793]
After rebuilding CCTBX on Crusher, importing a kokkos component like `from simtbx.kokkos import gpu_instance` causes errors complaining about undefined symbols:

```
In [1]: from simtbx.kokkos import gpu_instance
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
[...]
ImportError: __import__("simtbx_kokkos_ext"): /gpfs/alpine/chm137/proj-shared/wittwer/alcc-recipes/cctbx/build/lib/libsimtbx_kokkos.so: undefined symbol: __kmpc_for_static_init_8u
```

**Comment by nksauter** [comment]
Should be fixed by 0c19d90aed69f61f3b035829e94525c8997f20d3 committed Oct. 18.  @Tzrs are your tests working now?

**Comment by nksauter** [comment]
Should be fixed by 0c19d90aed69f61f3b035829e94525c8997f20d3 committed Oct. 18.  @Trzs are your tests working now?
